ui/components/app/modals/confirm-delete-network/confirm-delete-network.component.js

```diff
@@ -5,18 +5,18 @@ import Modal, { ModalContent } from '../../modal';
 export default class ConfirmDeleteNetwork extends PureComponent {
   static propTypes = {
     hideModal: PropTypes.func.isRequired,
-    removeNetworkConfiguration: PropTypes.func.isRequired,
+    removeNetwork: PropTypes.func.isRequired,
     onConfirm: PropTypes.func.isRequired,
-    target: PropTypes.string.isRequired,
     networkNickname: PropTypes.string.isRequired,
+    chainId: PropTypes.string,
```

Comment:
is this optional now?

Reply:
I'll add `.isRequired`

Reply:
https://github.com/MetaMask/metamask-extension/pull/26433/commits/b1e2c6530ed3bb8d213da3d41f1b98d8cfa28b05